File: B8_RestFragmentsAsync/WeatherReportApp/app/src/main/java/com/example/tsvetelinastoyanova/weatherreportapp/WeatherApplication.java
package com.example.tsvetelinastoyanova.weatherreportapp;

import android.app.Application;

import com.facebook.stetho.Stetho;

public class WeatherApplication extends Application {
    public void onCreate() {
        super.onCreate();
        Stetho.initializeWithDefaults(this);
    }
}

File: I1_MVPAndRetrofit_Kotlin/app/src/main/java/com/example/tsvetelinastoyanova/weatherapp/WeatherApplication.kt
package com.example.tsvetelinastoyanova.weatherapp

import android.app.Application
import com.facebook.stetho.Stetho

class WeatherApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Stetho.initializeWithDefaults(this)
    }
}
